eng/pipelines/runtimelab.yml

```diff
@@ -81,9 +62,9 @@ stages:
       buildConfig: Release
       platforms:
       - Linux_x64
-      - windows_x64
+      - FreeBSD_x64
       jobParameters:
-        timeoutInMinutes: 100
+        timeoutInMinutes: 120
         isOfficialBuild: ${{ variables.isOfficialBuild }}
         testGroup: innerloop
         extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
```

Comment:
Since you are not building tests, you probably want to set the `uploadLibrariesTests` parameter under `extraStepsParameters` to false always.

Reply:
That one I don't know. I would like to get to point where we _can_  run tests. It seems like we now have some test_assets but I did not have time to figure out if that is sufficient to just download and run on FreeBSD. That may be useful even if there is no Helix (yet?).

Reply:
Ok, I don't mind leaving that as is. However you definitely need to set `uploadIntermediateArtifacts` to `true`.

Reply:
should I set the `uploadIntermediateArtifacts` now? I certainly don't really know what exactly those settings do. 
For me the priority is:
1) get build rolling
2) publish and consume product Nuget package
3) get test running somehow

Reply:
In order to publish and consume NuGet package you need that setting to `true`. That is what publishes the NuGet packages into an artifact and then we download that artifact and send them to sign/publish.

Reply:
updated. BTW to what feed the packages would go?

Reply:
The intermediates did show up https://dev.azure.com/dnceng/_apis/resources/Containers/8362254/IntermediateArtifacts?itemPath=IntermediateArtifacts%2FFreeBSD_x64%2FShipping%2Fdotnet-runtime-6.0.0-ci-freebsd-x64.tar.gz

Reply:
> BTW to what feed the packages would go?

runtimelab official builds are publishing to https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet-experimental